src/main/java/forestry/core/EventHandlerCore.java

```diff
@@ -140,4 +145,16 @@ public void lootLoad(LootTableLoadEvent event) {
 			}
 		}
 	}
+	
+	@SubscribeEvent
+	public void handleVillagerAI(EntityJoinWorldEvent event) {	//won't register if villager not registered
+		IForgeRegistry<VillagerRegistry.VillagerProfession> villagerProfessions = ForgeRegistries.VILLAGER_PROFESSIONS;
+		Entity entity = event.getEntity();
+		if((entity instanceof EntityVillager)) {
+			EntityVillager villager = (EntityVillager) event.getEntity();
+			if(villagerProfessions.getKey(villager.getProfessionForge()).toString().equals(Constants.ID_VILLAGER_APIARIST)) {
```

Comment:
Please break this long line into smaller parts by storing things to variables before the `if`.